code/game/machinery/camera/camera_presets.dm

```diff
@@ -38,21 +38,24 @@
 /obj/machinery/camera/autoname
 	var/number = 0 //camera number in area
 
-//This camera type automatically sets it's name to whatever the area that it's in is called.
+// This camera type automatically sets it's name to whatever the area that it's in is called.
 /obj/machinery/camera/autoname/Initialize(mapload)
-	. = ..()
 	number = 1
 	var/area/A = get_area(src)
-	if(A)
-		for(var/obj/machinery/camera/autoname/C in GLOB.machines)
-			if(C == src)
-				continue
-			var/area/CA = get_area(C)
-			if(CA.type == A.type)
-				if(C.number)
-					number = max(number, C.number + 1)
-		c_tag = "[A.name] #[number]"
+	if(!A)
+		number = rand(1, 100)
```

Comment:
Might also want to put down a stacktrace here. I don't think this should be expected behaviour ever